### ISSUE TITLE
not compatible with 5.x.x

### ISSUE THREAD

**Issue by tedder** [lukewaite/logstash-input-cloudwatch-logs#20]
With a very minimal config and the official Logstash docker containers, this plugin isn't compatible with logstash 5.x. Here's the 5.0.0 error:

```
...
22:31:58.185 [[main]<cloudwatch_logs] ERROR logstash.pipeline - A plugin had an unrecoverable error. Will restart this plugin.
  Plugin: <LogStash::Inputs::CloudWatch_Logs log_group=>"staging5-docker", type=>"docker_cloudwatchlogs", id=>"96031aebedd7bbedf8fe2af725162a5ebb66f694-1", enable_metric=>true, codec=><LogStash::Codecs::Plain id=>"plain_bf72ffa9-05f4-47d5-8412-3e5af3e9f35e", enable_metric=>true, charset=>"UTF-8">, region=>"us-east-1", interval=>60, log_group_prefix=>false>
  Error: Direct event field references (i.e. event['field'] = 'value') have been disabled in favor of using event get and set methods (e.g. event.set('field', 'value')). Please consult the Logstash 5.0 breaking changes documentation for more details.
  Exception: NoMethodError
  Stack: /usr/share/logstash/logstash-core-event-java/lib/logstash/event.rb:44:in `method_missing'
/usr/share/logstash/vendor/bundle/jruby/1.9/gems/logstash-input-cloudwatch_logs-0.10.1/lib/logstash/inputs/cloudwatch_logs.rb:124:in `process_log'
...
```

That's in [the process_log function, line 127](https://github.com/lukewaite/logstash-input-cloudwatch-logs/blob/26fe6ce20504244e2f94767f2d2e495afe0ba2e3/lib/logstash/inputs/cloudwatch_logs.rb#L127). It looks like you changed those lines to `event.set` in 6e7cc5decdcd7a8d8528d42a7b040b1d2f3a3490, but removed it two weeks ago with the eb38dfdc072b4fd21e9c1d83ea306e2b6c5df37b merge.

Here's the error in 5.3.0:

```
[2017-04-20T22:41:40,344][ERROR][logstash.pipeline        ] A plugin had an unrecoverable error. Will restart this plugin.
  Plugin: <LogStash::Inputs::CloudWatch_Logs log_group=>"staging5-docker", type=>"docker_cloudwatchlogs", id=>"96031aebedd7bbedf8fe2af725162a5ebb66f694-1", enable_metric=>true, codec=><LogStash::Codecs::Plain id=>"plain_ab41f289-30d5-4f40-a07f-5a1895e071b5", enable_metric=>true, charset=>"UTF-8">, region=>"us-east-1", interval=>60, log_group_prefix=>false>```

**Comment by lukewaite** [comment]
Gah, yeah, bad merge. Let me get that fixed up and a new tag out.

**Comment by lukewaite** [comment]
@tedder Just published `v0.10.2` which should fix the events api issue. With luck that solves them all. 

We're not running Logstash 5 in production yet, so I'm a bit behind myself on the releases. Sorry about that!
